examples/source/1.components/amp-call-tracking.html

```diff
@@ -40,7 +40,7 @@
 
     We also need to set a default phone number that will be shown before the endpoint responds. This is done via an `<a>` element inside the `amp-call-tracking` tag.
   -->
-    <amp-call-tracking config="/static/samples/json/call-tracking-number.json">
+    <amp-call-tracking config="https://amp.dev/static/samples/json/call-tracking-number.json">
```

Comment:
Actually

```suggestion
    <amp-call-tracking config="<% hosts.platform %>/static/samples/json/call-tracking-number.json">
```

would be nice to be environment agnostic 🙂

Reply:
good point!